core/pom.xml

```diff
@@ -30,6 +30,10 @@
          <artifactId>jgroups</artifactId>
       </dependency>
 
+      <dependency>
```

Comment:
Interesting piece for review: Adding `KUBE_PING` to core, since we would like to support it out of the box.

There are some alternatives to this approach like:
* creating another module (infinispan-cloud?) with all additional discovery modules
* add other discovery modules only to Uber Jars